src/mscorlib/src/Microsoft/Win32/RegistryKey.cs

```diff
@@ -481,22 +318,6 @@ public unsafe string[] GetValueNames()
         }
 
         /**
-         * Retrieves the specified value. <b>null</b> is returned if the value
-         * doesn't exist.
-         *
-         * Note that <var>name</var> can be null or "", at which point the
-         * unnamed or default value of this Registry key is returned, if any.
-         *
-         * @param name Name of value to retrieve.
-         *
-         * @return the data associated with the value.
-         */
-        public Object GetValue(String name)
```

Comment:
Not from here. CoreFX has a completely independent and rewritten Registry implementation.